### ISSUE TITLE
BUG / Usage: local treeInfo = C_Traits.GetTreeInfo(configID, treeID)

### ISSUE THREAD

**Issue by tatatada** [Hekili/hekili#1861]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for this issue and was not able to find one.
- [X] I edited the title of this bug report (above) so that it describes the issue I am reporting.

### WoW Version

Retail (Dragonflight)

### Describe the Issue

5x Usage: local treeInfo = C_Traits.GetTreeInfo(configID, treeID)
[string "=[C]"]: ?
[string "@Interface/AddOns/Blizzard_SharedTalentUI/Blizzard_SharedTalentUtil.lua"]:253: in function `GetTreeInfo'
[string "@Interface/AddOns/Blizzard_SharedTalentUI/Blizzard_SharedTalentFrame.lua"]:919: in function `UpdateTreeInfo'
[string "@Interface/AddOns/Hekili/Dragonflight/Classes.lua"]:72: in function <Interface/AddOns/Hekili/Dragonflight/Classes.lua:66>

Locals:
(*temporary) = "Usage: local treeInfo = C_Traits.GetTreeInfo(configID, treeID)"

### How to Reproduce

just login in

### Player Information (Link)

```Text
build: v10.0.0-06.3
level: 60 (60)
class: SHAMAN
spec: restoration

talents: none

pvptalents: none

covenant: night_fae

conduits: condensed_anima_sphere = 1
   essential_extraction = 1
   focused_lightning = 1
   magma_fist = 1
   refreshing_waters = 1
   thunderous_paws = 1

soulbinds: [niya]
   bonded_hearts = 1
   called_shot = 1
   grove_invigoration = 1
   niyas_tools_poison = 1
   run_without_tiring = 1
   swift_patrol = 1

sets: tuft_of_smoldering_plumage = 1

gear: armschienen_des_theurgischen_sternensprechers = 1
    beflecktes_insigne_von_quelthalas = 1
    bschel_schwelendes_gefieder = 1
    elegie_der_ewigen = 1
    grtel_des_theurgischen_sternensprechers = 1
    gugel_des_theurgischen_sternensprechers = 1
    knochenschmetternde_stulpen = 1
    modisches_unterhemd = 1
    ringpanzer_des_theurgischen_sternensprechers = 1
    sabatons_des_theurgischen_sternensprechers = 1
    schal_des_theurgischen_sternensprechers = 1
    schattenschreckring = 1
    tassetten_des_theurgischen_sternensprechers = 1
    tuft_of_smoldering_plumage = 1
    verfluchtes_opus = 1
    wappenrock_der_natur = 1
    weltenvernichtende_iris = 1
    zier_des_theurgischen_sternensprechers = 1

legendaries: seeds_of_rampant_growth = 1

itemIDs: 151116, 172324, 178926, 184020, 186377, 186406, 188270, 188919, 188920, 188921, 188922, 188923, 188924, 188926, 188927, 189859, 38309

settings: none

toggles: cooldowns = false 
    custom1 = false 
    custom2 = false 
    defensives = false 
    essences = true 
    interrupts = false 
    mode = automatic 
    potions = false 

keybinds: 

warnings: none
```

### Error Messages (Link)

```Text
5x Usage: local treeInfo = C_Traits.GetTreeInfo(configID, treeID)
[string "=[C]"]: ?
[string "@Interface/AddOns/Blizzard_SharedTalentUI/Blizzard_SharedTalentUtil.lua"]:253: in function `GetTreeInfo'
[string "@Interface/AddOns/Blizzard_SharedTalentUI/Blizzard_SharedTalentFrame.lua"]:919: in function `UpdateTreeInfo'
[string "@Interface/AddOns/Hekili/Dragonflight/Classes.lua"]:72: in function <Interface/AddOns/Hekili/Dragonflight/Classes.lua:66>

Locals:
(*temporary) = "Usage: local treeInfo = C_Traits.GetTreeInfo(configID, treeID)"
```

**Comment by Hekili** [comment]
This should be fixed in 10.0.0-06.4.

**Comment by tatatada** [comment]
thank u very much for your efforts. like your addon alot!
